packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -257,6 +257,7 @@ const SettingsSidebarContainer = ({
 
   const isOrgAdminOrOwner =
     currentOrg && currentOrg?.user?.role && ["OWNER", "ADMIN"].includes(currentOrg?.user?.role);
+  let bottomMargin = false;
```

Comment:
Why is this needed?

Reply:
Margin was only being applied on the last child of the Tab

https://github.com/calcom/cal.com/blob/ae64d7ba8223c2d2992a4db9f003a1c94efa869f/packages/features/settings/layouts/SettingsLayout.tsx#L315-L327

We set it to true of in case of Org Members to also add margin there

https://github.com/calcom/cal.com/blob/fb92d0c827bddf06c639e6c9831a631da3d187be/packages/features/settings/layouts/SettingsLayout.tsx#L322-L324

https://github.com/calcom/cal.com/blob/fb92d0c827bddf06c639e6c9831a631da3d187be/packages/features/settings/layouts/SettingsLayout.tsx#L333-L335

![Screenshot 2024-04-16 at 10 07 49 PM](https://github.com/calcom/cal.com/assets/74371312/22043827-91f6-42ff-bf33-dc9446c5a23f)